docs/admin/maintenance/noble_migration.rst

```diff
@@ -5,18 +5,11 @@ SecureDrops need to be upgraded to the newer Ubuntu 24.04 (Noble)
 operating system. This process is far simpler than past upgrades
 as it has been fully automated.
 
-Administrators have two options, on the following timeline:
+Begnning April 22nd, 2025, the SecureDrop team began automatically triggering the upgrade process, in phases, across all SecureDrop instances.
 
-* **semiautomated, through April 15, 2025:** Administrators can manually trigger the upgrade and observe the process.
-* **fully automated, after April 15, 2025:** The SecureDrop team will push updates through mid- to late-April that automatically trigger the upgrade process, in phases, across all SecureDrop instances.
+If your SecureDrop has not yet been selected for an automated upgrade, you can still manually trigger a semiautomated upgrade to force the upgrade to Noble. We still recommend the semiautomated upgrade for larger instances or if you have a non-standard setup, as the upgrade will happen whenever you choose so you will already be available in case something goes wrong during the process.
 
-The fully automated upgrade is the simplest option, as it requires no action on your part.
-
-We recommend the semiautomated upgrade for larger instances or if you have a non-standard setup as
-the upgrade will happen whenever you choose it, so you will already be available in case something goes
-wrong during the process.
-
-.. warning:: If you are using not using `recommended hardware <../installation/hardware.html#specific-hardware-recommendations>`_ for your *Application* and *Monitor Servers*, we strongly recommend doing a semiautomated upgrade **before** April 15th, 2025. 
+.. warning:: If you are using not using `recommended hardware <../installation/hardware.html#specific-hardware-recommendations>`_ for your *Application* and *Monitor Servers*, we strongly recommend doing a semiautomated upgrade if your server has not yet completed an automated upgrade.
```

Comment:
Same, we're not doing mon yet, so I think we should still recommend this and don't need to add the "if your server has not yet completed an automated upgrade".